addons/website_portal/controllers/main.py

```diff
@@ -9,6 +9,16 @@
 from odoo.fields import Date
 
 
+def get_records_pager(ids, current):
```

Comment:
because it is easier to save ids in session that un serializable browse record.
So yes, ids are the list of ids in the history, and record, is the current record... I thniks that the name are explicit...